docs/topics/Gateway.md

```diff
@@ -96,7 +97,7 @@ def on_websocket_message(msg):
 ###### Gateway URL Params
 
 | Field     | Type    | Description                                   | Accepted Values                                                            |
-| --------- | ------- | --------------------------------------------- | -------------------------------------------------------------------------- |
+|-----------|---------|-----------------------------------------------|----------------------------------------------------------------------------|
 | v         | integer | Gateway Version to use                        | 6 (see [Gateway versions](#DOCS_TOPICS_GATEWAY/gateways-gateway-versions)) |
```

Comment:
```suggestion
| v         | integer | Gateway Version to use                        | see [Gateway versions](#DOCS_TOPICS_GATEWAY/gateways-gateway-versions) |
```
Instead of duplicating this documentation, probably easier just to link back to the table you have above :)